src/app/legacy/components/redeem-widget/use-formatted-data.tsx

```diff
@@ -35,12 +36,26 @@ export function useFormattedData() {
     [inputTokenAmount, balance],
   )
 
-  const outputAmount = quote?.outputTokenAmount
-    ? `${formatAmount(Number(formatWei(quote?.outputTokenAmount.toBigInt(), quote?.outputToken.decimals)))} ${quote?.outputToken.symbol}`
-    : ''
-  const outputAmountUsd = quote?.outputTokenAmountUsd
-    ? `$${formatAmount(quote?.outputTokenAmountUsd)}`
-    : ''
+  let outputAmounts: string[] = []
+  let outputAmountsUsd: string[] = []
+  let outputAmountUsd = ''
+  if (legacyQuote) {
+    const { outputTokens, outputTokenPricesUsd, units } = legacyQuote
+    outputAmounts = outputTokens.map(
+      (outputToken, index) =>
+        `${formatAmount(Number(formatWei(units[index], outputToken.decimals)))}`,
```

Comment:
nitpick: correct me if there is a reason for this, but I think formatAmount already returns string, and since there isn't anything just the value in the template literal, its unnecessary I think

Reply:
good spot. deleted something during development and didn't update 👌